pyproject copy.toml

```diff
@@ -0,0 +1,64 @@
+[tool.poetry]
```

Comment:
I'm not sure what the `pyproject copy.toml `file is for? It looks the same as `pyproject.toml` so assuming this was just an accidental include?

Reply:
Yes, that was totally accidental. :)
Removed.